src/Mono.Android/Android.Runtime/JNINativeWrapper.cs

```diff
@@ -19,7 +19,7 @@ static void get_runtime_types ()
 			mono_unhandled_exception_method = typeof (System.Diagnostics.Debugger).GetMethod (
 				"Mono_UnhandledException", BindingFlags.NonPublic | BindingFlags.Static);
 			if (mono_unhandled_exception_method == null)
-				AndroidEnvironment.FailFast ("Cannot find System.Diagnostics.Debugger.Mono_UnhandledException");
+				//AndroidEnvironment.FailFast ("Cannot find System.Diagnostics.Debugger.Mono_UnhandledException");
```

Comment:
This is also called here:

https://github.com/xamarin/xamarin-android/blob/d48cf04f9749664bf48fc16bcb920d5d941cccab/src/Mono.Android/Android.Runtime/JNIEnv.cs#L242-L279

@marek-safar / @steveisok is there an equivalent for `System.Diagnostics.Debugger.Mono_UnhandledException`?

We think this is what tells the debugger to break in the IDE if an unhandled exception is thrown.

Reply:
I think @lambdageek commented on that in XI doc. Do you know why is it needed?

/cc @thaystg

Reply:
I don't think we have anything similar for XI.
This is used for trigger the debugged process to send an unhandled exception message to the agent. This is needed since MonoDroid can't know at the Java->C# boundary if a given C# exception is unmanaged or not.